### DISCUSSION TITLE
Can not restore Nextcloud AIO from backup

### DISCUSSION THREAD

**Discussion by nikolaos70** [nextcloud/all-in-one#4120] [Questions] [answered]
Please help me solve the problem. When I try to restore a Nextcloud AIO from a Borg backup I get:

Restoring '20240126_131903-nextcloud-aio'...
fuse: mount failed: Permission denied
Could not mount the backup!

All permissions are checked. What could be wrong?

**Comment by szaimen** [comment]
Hi, is the backup archive maybe positioned on a NFS share? 

Also is SELinux enabled on your host?

**Reply by nikolaos70** [reply]
No. The backup archive on the /var/backup/borg directory on the host (VM Ubuntu 22). SELinux is not enabled. I tried to set chown -R www-data:www-data and chmod -R 777 of the directory and so on. But nothing helps. 

I don't understand what mount is failed. On the host? Or inside the docker image? Which permission could be denied? If you can explain that i I'm so grateful.

**Reply by szaimen** [reply]
Can you post the output of `sudo docker info` here?

**Reply by nikolaos70** [reply]
Client: Docker Engine - Community
 Version:    24.0.6
 Context:    default
 Debug Mode: false
 Plugins:
  buildx: Docker Buildx (Docker Inc.)
    Version:  v0.11.2
    Path:     /usr/libexec/docker/cli-plugins/docker-buildx
  compose: Docker Compose (Docker Inc.)
    Version:  v2.21.0
    Path:     /usr/libexec/docker/cli-plugins/docker-compose

Server:
 Containers: 12
  Running: 12
  Paused: 0
  Stopped: 0
 Images: 12
 Server Version: 24.0.6
 Storage Driver: overlay2
  Backing Filesystem: extfs
  Supports d_type: true
  Using metacopy: false
  Native Overlay Diff: true
  userxattr: false
 Logging Driver: json-file
 Cgroup Driver: systemd
 Cgroup Version: 2
 Plugins:
  Volume: local
  Network: bridge host ipvlan macvlan null overlay
  Log: awslogs fluentd gcplogs gelf journald json-file local logentries splunk syslog
 Swarm: inactive
 Runtimes: io.containerd.runc.v2 runc
 Default Runtime: runc
 Init Binary: docker-init
 containerd version: 8165feabfdfe38c65b599c4993d227328c231fca
 runc version: v1.1.8-0-g82f18fe
 init version: de40ad0
 Security Options:
  apparmor
  seccomp
   Profile: builtin
  cgroupns
 Kernel Version: 5.15.0-91-generic
 Operating System: Ubuntu 22.04.3 LTS
 OSType: linux
 Architecture: x86_64
 CPUs: 1
 Total Memory: 2.801GiB
 Name: ubuntu-2204
 ID: 15117edf-396c-44cd-8687-c963c09c81e4
 Docker Root Dir: /var/lib/docker
 Debug Mode: false
 Experimental: false
 Insecure Registries:
  127.0.0.0/8
 Live Restore Enabled: false

**Comment by szaimen** [comment]
Hi, I found the bug. It will be fixed as part of the next release. For tracking: https://github.com/nextcloud/all-in-one/issues/4124

**Reply by szaimen** [reply, marked as answer]
This is now fixed with v7.11.2 Beta. Testing and feedback is welcome! See https://github.com/nextcloud/all-in-one#how-to-switch-the-channel

**Comment by nikolaos70** [comment]
The restore is ok! Thank You very much!
